sql/hive/src/test/scala/org/apache/spark/sql/catalyst/LogicalPlanToSQLSuite.scala

```diff
@@ -45,7 +45,7 @@ class LogicalPlanToSQLSuite extends SQLBuilderTest with SQLTestUtils {
 
   // Used for generating new query answer files by saving
   private val regenerateGoldenFiles: Boolean = System.getenv("SPARK_GENERATE_GOLDEN_FILES") == "1"
-  private val goldenSQLPath = getTestResourcePath("sqlgen")
+  private val goldenSQLPath = "src/test/resources/sqlgen/"
```

Comment:
Hi, @rxin . 
IMO, this bug on `LogicalPlanToSQLSuite` is needed to be fixed soon. If this PR is needed to be reviewed more, should I make another PR containing that only?